product_docs/docs/edbcloud/beta/getting_started/02_connect_cloud_account.mdx

```diff
@@ -29,12 +31,50 @@ Take note of the **Application (client) ID**, you need it to configure your EDB
 1. Choose _application secret_ as an authentication option for the application. See [Create a new Azure AD application secret](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret) for instructions. Take note of the Azure AD App Secret, you need it to configure your cloud account.
 1. Assign the owner role to the application. See [Assign a role to the application](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#assign-a-role-to-the-application) for instructions. Enter the **Display name** of the Azure AD application in the **Select** field of the **Add role assignment** panel. See [Open the Add role assignment pane](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current#step-2-open-the-add-role-assignment-pane) for instructions.
 
-
-
-
-
-
-
+### Create Azure Active Directory Application Using `create-spn`
+
+To simplify the process of creating an Azure AD Application, EDB provides the `create-spn` script for Azure API users. You can download it [here](https://github.com/EnterpriseDB/cloud-utilities/tree/main/azure).
+Before using the script ensure that the following utilities are installed on your machine:
+* [jq command-line JSON processor](https://stedolan.github.io/jq/)
+* [azure cli](https://docs.microsoft.com/en-us/cli/azure/) v2.26 or above
+
+#### create-spn Usage
```

Comment:
I don't think we need this block at all - its information is completely described in the preceding and succeeding sections.